### ISSUE TITLE
asyncSeq not available

### ISSUE THREAD

**Issue by KingKnecht** [fsprojects/FSharp.Control.AsyncSeq#133]
### Description

Neither `asyncSeq `is available nor `AsyncSeq`

![image](https://user-images.githubusercontent.com/4339172/104133162-13531d00-5382-11eb-8380-374a76499407.png)

### Repro steps

1. Create a new F# console application (netcoreApp3.1)
2. paket init
3. paket add FSharp.Control.AsyncSeq (paket version 5.257.0, FSharp.Control.AsyncSeq version 3.0.3)
4. Add reference to project file
Project file looks like:

```
<?xml version="1.0" encoding="utf-8"?>
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>netcoreApp3.1</TargetFramework>
  </PropertyGroup>
  <ItemGroup>
    <Compile Include="Program.fs" />
  </ItemGroup>
  <Import Project=".paket\Paket.Restore.targets" />
</Project>
```
6. open FSharp.Control in Program.fs
7. try to use asyncSeq

### Expected behavior

AsyncSeq should be available.

### Actual behavior

AsyncSeq is missing

![image](https://user-images.githubusercontent.com/4339172/104133219-78a70e00-5382-11eb-8f97-42bf532feeab.png)

### Related information 

* Operating system -> Win10 x64
* .NET Runtime -> 3.1

Am I missing something here? Let me know if you need more information.
Thx.

**Comment by wilsoncg** [comment]
I created a fresh console project from scratch and noticed similar behaviour.  It's quickly resolved with a `dotnet build` and a reload window command (ctrl+shit+P, `reload window`).

**Comment by wilsoncg** [comment]
```xml
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>netcoreapp3.1</TargetFramework>
  </PropertyGroup>

  <ItemGroup>
    <Compile Include="Program.fs" />
  </ItemGroup>

  <ItemGroup>
    <PackageReference Include="FSharp.Control.AsyncSeq" Version="3.0.3" />
  </ItemGroup>

</Project>
```

```fsharp
// Learn more about F# at http://docs.microsoft.com/dotnet/fsharp

open System
open FSharp.Control

// Define a function to construct a message to print
let from whom =
    sprintf "from %s" whom

let a = asyncSeq {
    yield 1 
}
let b = a |> AsyncSeq.toAsyncEnum

[<EntryPoint>]
let main argv =
    let message = from "F#" // Call the function
    printfn "Hello world %s" message
    0 // return an integer exit code
```

![image](https://user-images.githubusercontent.com/1066103/104184383-48f01880-540b-11eb-9da6-57cea739caf4.png)

**Comment by dsyme** [comment]
Hmmm strange, I haven't tried to reproduce.  

Are there any warnings on command line restore of build using `dotnet restore` or `dotnet build`?  Perhaps post a log of a clean build using `dotnet build -v:d`

**Comment by KingKnecht** [comment]
Tested it on another machine today. At first, same behavior. Then I did a couple of "Reload window", which I didn't know is available. Now `AsyncSeq` is there. 
That's strange, because I didn't know the "Reload window" command, I always closed & opened VSCode, which didn't fix it.
In my "real" project I'm using FAKE (it's a SAFE Stack based app). So I'm not sure if `dotnet build` would make much sense here. Maybe for the server part?
